Gemfile

```diff
@@ -19,7 +19,6 @@ gem 'jbuilder', '~> 2.11' # Standard part of Rails, but unused, since we don't h
 gem 'pg', '~> 1.2' # PostgreSQL support.
 gem 'pg_search', '~> 2.3' # builds ActiveRecord named scopes that take advantage of PostgreSQL's full text search.
 gem 'activerecord-precounter', '~> 0.4' # N+1 count query optimizer for ActiveRecord.
-gem 'sass-rails', '>= 6'
```

Comment:
@harigopal - Bit confused on the implementation with `roadie-rails` here. Even though we are not using roadie-rails for custom CSS, but still this is a dependency for sending emails in [school_mailer ](https://github.com/pupilfirst/pupilfirst/blob/aae494620bde2e0c8d6f77d6c94dd02c1e178a95/app/mailers/school_mailer.rb#L38) 🤔 Should we update this method to use `mail` instead of `roadie_mail`

Reply:
@prashanth726 Please look into the documentation of the roadie-rails gem. There's nothing special about how it's used in the mailers - we're simply following instructions from the gem: https://github.com/Mange/roadie-rails#manual-usage

My question was whether this gem is still required. Removal of  the gem would, of course, mean that its associated usage will also need to be removed. The question is whether this full removal would affect anything.